scripts/osrm-runner.js

```diff
@@ -123,11 +118,8 @@ queries = queries.map(q => { return {hostname: options.server.hostname, port: op
 http.globalAgent.maxSockets = options['max-sockets'];
 queries.map(query => {
     run_query(query, options.filter, (query, code, ttfb, total, results) => {
-        let str = `"${query}",${code}`;
-        if (ttfb !== undefined) str += `,${ttfb}`;
-        if (total !== undefined) str += `,${total}`;
-        if (typeof results === 'object' && results.length > 0)
-            str += ',' + results.map(x => isNaN(x) ? '"' + JSON.stringify(x).replace(/\n/g, ';').replace(/"/g, "'") + '"' : Number(x)).join(',');
-        console.log(str);
+        let data = results ? JSON.stringify(results[0]) : '';
+        let record = [[query, code, ttfb, total, data]];
+        process.stdout.write(csv_stringify(record));
```

Comment:
looks good and clean!
this changes the behaviour a bit:
if one of the records doesnt exist (e.g. `query`, `code`)
    - the new code will emit a `,` and an empty string
    - the old code does not emit a `,`

i think the new code is better, but just wanted to make sure we are aware that the behaviour changes.

the old code is especially dangerous if `total` would exist but `ttfb` doesn't. then the commas and the positions are off. so new behaviour is better! 👍